NSURLSessionCancellationSwift.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |spec|
+  spec.name = 'NSURLSessionCancellationSwift'
```

Comment:
In the Swift 3.0 migration the extension was migrated to URLSession (since that's how Cocoa is imported now in Swift 3.0 projects). Do you think we should rename the source file and this Pod spec name? A tricky one if we want to backport the Pod to Swift 2.3 then I think we should keep the name otherwise we have the option to rename. What do you think?